### ISSUE TITLE
Comparison with original GCN

### ISSUE THREAD

**Issue by wzm2256** [danielegrattarola/keras-gat#13]
I'm comparing this  code with original GCN, is there any way I can silence the ''attention'' tensor (which I believe is the softmax tensor in class GraphAttention, right?) and keep the other setting fixed?

So that if your code behave differently with and without ''attention'' mechanism, we can say ''attention'' works.

**Comment by danielegrattarola** [comment]
You might want to have a look at #17. 
In short, attention works, but GCN can be almost as effective if you add dropout and bias.
